### DISCUSSION TITLE
Cannot install gatsby

### DISCUSSION THREAD

**Discussion by tjddbs0401** [gatsbyjs/gatsby#34436] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

` npm install -g gatsby-cli ` 
doesn't work: 6 warnings appear, but there isn't any error message.
But I cannot use gatsby after installing.

<img width="694" alt="스크린샷 2022-01-08 오후 8 19 47" src="https://user-images.githubusercontent.com/48537546/148642205-81961107-b58e-4952-b488-a5cfb5108049.png">

### Reproduction Link

x

### Steps to Reproduce

Not reproducible.

### Expected Result

Run gatsby

### Actual Result

npm WARN deprecated @hapi/topo@3.1.6: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated @hapi/bourne@1.3.2: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated @hapi/address@2.1.4: Moved to 'npm install @sideway/address'
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated @hapi/hoek@8.5.1: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated @hapi/joi@15.1.1: Switch to 'npm install joi'

changed 575 packages, and audited 576 packages in 11s

108 packages are looking for funding
  run `npm fund` for details

8 vulnerabilities (3 moderate, 5 high)

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.

>gatsby info
zsh: command not found: gatsby

### Environment

```shell
OS: macOS Catalina 10.15.7
CPU: 1.6 GHz dual-core Intel Core i5
node -v: v16.13.1
npm: 8.1.2
```

**Comment by AdrianGuery** [comment]
Try to update your config ([source](https://stackoverflow.com/questions/52378823/gatsby-version-bash-gatsby-command-not-found))

`npm config delete prefix`

`npm config set prefix /usr/local`

`npm i -g gatsby-cli`

`gatsby --version`
